src/components/app/app.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import WelcomeScreen from "../welcome-screen/welcome-screen.jsx";
+
+const App = (props) => {
```

Comment:
```suggestion
const App = ({errorsCount}) => {
```

Ты же на втором даже так писала. Что мешает тут так писать?